src/views/communitySettings/components/editForm.js

```diff
@@ -301,6 +311,13 @@ class EditForm extends React.Component<Props, State> {
         <SectionTitle>Community Settings</SectionTitle>
         <Form onSubmit={this.save}>
           <ImageInputWrapper>
+            {coverPhoto && !/default_images/.test(coverPhoto) && (
```

Comment:
I've never seen this `!/default_images/.test(coverPhoto)` syntax before - could you point me to where I can learn what's going on here?

Reply:
It's a regex method to check if there's a default image or custom (user uploaded image) and to show or hide the delete (X) button.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test

Reply:
Thank you!